Probesys/bileto
# new: Open tickets via the interface

Related to https://github.com/Probesys/bileto/issues/42

Changes proposed in this pull request:

- Add a `uid` column to the organization entity (generated with a new `Random` class)
- Create the Ticket entity (+ migration + repository + factory)
- Add a form to create tickets
- Add a page to show a ticket
- Add a very basic list of tickets of organizations

How to test the feature manually:

0. Note that you need to reset your database with `make db-reset FORCE=true`
1. Create an organization
2. Open a ticket in the organization
3. Check it works and that the main information of the ticket are displayed

The main restriction is that we cannot add a message to detail the ticket. It will be done soon after this PR. I'll also add colors to the status and priority badges in a future PR (tickets to be created).

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
